README.md

```diff
@@ -17,6 +17,28 @@ To get started with this sample, you have two options:
   [Quickstart: Add sign-in with Microsoft to a Python web app](https://docs.microsoft.com/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python).
 * Use PowerShell scripts that automatically create the Azure AD applications and related objects (passwords, permissions, dependencies) for you, and then modify the configuration files. Follow the steps in the [App Creation Scripts README](./AppCreationScripts/AppCreationScripts.md).
 
+# Deployment
+
+Once you finish testing this web app locally, you can deploy it to your production.
+You may choose any web app hosting services you want.
+Here we will describe how to deploy it to
+[Azure App Service](https://azure.microsoft.com/en-us/products/app-service).
+
+* Follow the ["Quickstart: Deploy a Python (Django or Flask) web app to Azure App Service"](https://learn.microsoft.com/en-us/azure/app-service/quickstart-python),
+  but replace its sample app (which does not do user sign-in) with this web app.
+
+* In particular, if you choose to ["deploy using Local Git" in "step 3 - Deploy your application code to Azure"](https://learn.microsoft.com/en-us/azure/app-service/quickstart-python?tabs=flask%2Cwindows%2Cazure-cli%2Clocal-git-deploy%2Cdeploy-instructions-azportal%2Cterminal-bash%2Cdeploy-instructions-zip-azcli#3---deploy-your-application-code-to-azure),
+  an [application-scope credential](https://learn.microsoft.com/en-us/azure/app-service/deploy-configure-credentials?tabs=portal#appscope)
+  will be automatically created with the shape as `your_app_name\$your_app_name`.
+  But your actual git username is only the `$your_app_name` part.
+
+* Do not forget to setup
+  [several environment variables](https://github.com/Azure-Samples/ms-identity-python-webapp/blob/master/.env.sample)
+  needed by this sample.
+  Otherwise your website will usually render a blank "Internal Error" page
+  (Flask's debug mode output does not seem to be relayed by App Service).
+
+

```

Comment:
The debug mode output is not relayed as it's bad practice to set FLASK_DEBUG to True on production. In the case of "Internal Error", developers can check the Log Stream for details.

Reply:
Yeah, I figure they strip out all the error page content for security concern, and rightfully so.

Log Stream seems to contain deployment logs only, but no app level logs. Presumably because [Log Stream only captures output on stdout](https://learn.microsoft.com/en-us/azure/app-service/quickstart-python?tabs=flask%2Cwindows%2Cazure-cli%2Cvscode-deploy%2Cdeploy-instructions-azportal%2Cterminal-bash%2Cdeploy-instructions-zip-azcli#5---stream-logs). Who would write print() in a server side app? Our app does not.

Reply:
I believe we could configure it to get logger.info() to output to Log Stream - but the default for Flask is likely warning level (vs info level). Instructions here:
https://flask.palletsprojects.com/en/2.2.x/logging/
Have you tried logging.warn()? Or app.logger.warn()?

Reply:
Fair point. I tried app.logger.error() just now. It technically works, and my log made it into the Log Stream. But it is not really helpful. It turns out, the Log Stream is an ever-increasing text page that combining *many* log files in the log folder. My error log line is just one line among other 500 lines in the stream, and my log line is not even guaranteed to be the last line being shown.

I still think that inline, to-the-point, error message inside our index() is much more helpful. Its discoverability is superb. First-time app service users may not even find the page for Log Stream, but they won't miss out an error served on their web app's index page.

Reply:
Lol, Log Stream is pretty janky right now, I tell the App Service team that every week- they say an improvement is coming! When I need legit logs that aren't strangely ordered, I go to Advanced Tools and download the Docker logs.

Anyway, yep, I like your idea, I just want it to be a template. Made change in my PR: https://github.com/Azure-Samples/ms-identity-python-webapp/pull/98